### PR TITLE
Fix: Made changes according to suggestions

### PR DESCRIPTION
**Pull request by zoef-yt**
Made changes according to suggestions by @zoyatanweer

- Changed 'Find products->' button location.
- Added a little arrow in the drop-down of filters.
- Added modal on user profile hover.
- Added placeholder to the textfield.

### CODE REVIEW

File: screens/cart/cart.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			<div class="navbar-brand">
 				<a href="../../index.html"> <h1>Need Games</h1> </a>
```
Comment (iRohitGaur) [inline screens/cart/cart.html]: you can do "/index.html" instead

File: screens/home-page/home-page.css [hunk 2]
```diff
@@ -62,7 +62,7 @@
 	width: fit-content;
 }
 
-.navbar-cta > *:hover {
+.navbar-cta > *:hover:not(:last-child) {
 	transform: translateY(-0.2rem);
```
Comment (iRohitGaur) [inline screens/home-page/home-page.css]: this should be in px

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge with updates

File: screens/cart/cart.html (hunk 1 above)
Comment (zoef-yt) [reply screens/cart/cart.html]: Will make the changes

**Review by Omkar-Ghate — APPROVED** [review]
Great UI. Good to merge.

File: screens/product-listing-page/product-listing-page.css [hunk 3]
```diff
@@ -124,3 +135,47 @@
...
+	right: 1rem;
+	margin-left: -5px;
+	border-width: 10px;
+	border-style: solid;
+	border-color: transparent transparent var(--colour-accent-third) transparent;
+}
+
+.profile-modal > * {
+	margin: 0.5rem;
+	border-radius: 0.5rem;
```
Comment (zoyatanweer) [inline screens/product-listing-page/product-listing-page.css]: maybe use pixels in both these?

File: screens/product-listing-page/product-listing-page.css [hunk 4]
```diff
@@ -124,3 +135,47 @@
...
+	transition: all 0.3s ease-in-out;
+}
+
+.profile-modal li:hover {
+	padding-left: 0.5rem;
```
Comment (zoyatanweer) [inline screens/product-listing-page/product-listing-page.css]: here too use pixels

**Review by zoyatanweer — APPROVED** [review]
clean code. good to merge.

File: screens/product-listing-page/product-listing-page.css (hunk 3 above)
Comment (bhushannaikaroskar) [reply screens/product-listing-page/product-listing-page.css]: yup use pixels here

File: screens/product-listing-page/product-listing-page.css [hunk 5]
```diff
@@ -41,6 +41,17 @@
 	z-index: 1;
 }
 
+.dropdown-content::after {
+	content: '';
+	position: absolute;
+	top: 0;
+	left: 45%;
+	margin-left: -5px;
+	border-width: 10px;
```
Comment (bhushannaikaroskar) [inline screens/product-listing-page/product-listing-page.css]: here use in rems as it is 10px.

**Review by bhushannaikaroskar — APPROVED** [review]
Some minor changes. rest good to merge

**Review by j24m — APPROVED** [review]
Code looks good to merge. And great work on the wishlist cards .

**Pull request merged**